pyproject.toml

```diff
@@ -9,7 +9,7 @@ description = "QuantLib extensions for Moscow Exchange (MOEX)"
 authors = [{ name = "Sergey Kostrukov", email = "sergey.dev@kostrukov.com" }]
 license = { file = "LICENSE" }
 requires-python = ">=3.9"
-dependencies = ["QuantLib==1.38.*"]
+dependencies = ["QuantLib==1.38.0"]
```

Comment:
Are you sure it needs to be `QuantLib==1.38.0` and not `QuantLib==1.38` ?  The official version format for the library seems to be 1.38